src/plus/webviews/graph/graphWebview.ts

```diff
@@ -113,76 +116,78 @@ export class GraphWebview extends WebviewWithConfigBase<State> {
 			return undefined;
 		}
 
-		const [currentUser, log, /*remotes,*/ tags, branches] = await Promise.all([
```

Comment:
@miggy-e FYI: I have moved that code because it should be in the render side (`GraphWrapper.tsx` file). 
We had already a prepared function to format the data to the GK graph model (see `getGraphModel` in `GraphWrapper.tsx` file).